pages/index.tsx

```diff
@@ -21,555 +24,282 @@ const LandingPage: FunctionComponent = () => {
-            analytics release workflow
-          </b>
-          . You can adopt all three, or stop at 1 (no engineering effort) or 2
-          (little to no engineering effort).
+          Below are the key areas to consider when getting started with Avo. You
+          can start them in any order or in parallel, and skip those you don't
+          think apply to you.
         </p>
 
-        <div className={styles.tocInline}>{tocContent}</div>
+        {headings != null && headings.length !== 0 && (
+          <div className={styles.inlineToc}>
+            <Toc headings={headings} />
+          </div>
+        )}
 
-        <h2 id="stage1" className={[styles.label, styles.heading2].join(' ')}>
-          Step 1. Tracking plan management – no engineering required
+        <h2 id="workflow" className={[styles.label, styles.heading2].join(' ')}>
+          Try the Avo workflow for your next analytics release
         </h2>
         <p className={styles.text}>
-          Use Avo instead of your spreadsheet to manage and discuss your
-          tracking plan changes. Avo is designed for analytics, so it&apos;s way
-          better than the spreadsheet for managing your tracking plan.
+          If you have an analytics release coming up in the next few days, we
+          highly recommend you try out the Avo Workflow for that release.
         </p>
-
         <p className={styles.text}>
-          This adoption step does not require any developer input, so you can
-          adopt it in the data design team without getting a developer buy-in.
+          The Avo workflow covers all the steps required to plan, review,
+          request implementation, implement, validate and publish your analytics
+          release.
         </p>
-        <h3 className={[styles.label, styles.heading3].join(' ')}>
-          Creating tracking plan in Avo
-        </h3>
+        <p className={styles.callout}>
+          <span>💡</span>
+          When trying the Avo Workflow for the first time, don't choose a
+          product release that's about to take the next 3 months to work on.
+          Choose one that has a handful of new or updated events.
+        </p>
+
+        <Zoom>
+          <img
+            className={styles.image}
+            width="100%"
+            src={require('../images/avo-workflow.png')}
+            alt="The Avo Workflow: Plan, review, implement, validate and publish your tracking changes"
+          />
+        </Zoom>
+
         <p className={styles.text}>
-          If you already have a tracking plan you can import it to Avo, or you
-          can create your plan in Avo from scratch. In any case you&apos;ll be
-          able to fine tune it and add new events for every feature release in
-          Avo.
+          We recommend your first “full-circle” release in Avo be a small one.
+          This will enable you to quickly go through your first analytics
+          release in Avo, from start to finish.
```

Comment:
Do we want to repeat the callout content here?

Reply:
Merge some of the copy from the callout above with this one in ba32008f3b69336d83c8f207795f95c52796f753, let me know what you think @hugihlynsson!